documents/document_session.go

```diff
@@ -7,17 +7,42 @@ import (
 )
 
 type IDocumentSession interface{
+	OpenSession() error
```

Comment:
`OpenSession()` is a method on `IDocumentStore` interface, not `IDocumentSession`.